src/target/llvm/llvm_instance.cc

```diff
@@ -153,6 +176,21 @@ LLVMTarget::LLVMTarget(LLVMInstance& instance, const Target& target)
     }
   }
 
+  if (const Optional<Array<String>>& v = target->GetAttr<Array<String>>("cl-opt")) {
+    llvm::StringMap<llvm::cl::Option*>& options = llvm::cl::getRegisteredOptions();
+    for (const String& s : v.value()) {
+      Option opt = ParseOptionString(s);
+      if (opt.type == Option::OptType::Invalid) {
+        continue;
+      }
+      if (options.count(opt.name)) {
+        llvm_options_.push_back(opt);
+      } else {
+        LOG(WARNING) << "\"" << opt.name << "\" is not an LLVM option, option ignored";
```

Comment:
should we error instead? same question above on line 184

Reply:
Sure, I can change the warnings to errors.